tstyche.config.json

```diff
@@ -1,19 +1,3 @@
 {
-  "testFileMatch": [
-    "packages/expect/__typetests__/*.test.ts",
-    "packages/expect-utils/__typetests__/*.test.ts",
-    "packages/jest/__typetests__/*.test.ts",
-    "packages/jest-cli/__typetests__/*.test.ts",
-    "packages/jest-expect/__typetests__/*.test.ts",
-    "packages/jest-mock/__typetests__/*.test.ts",
-    "packages/jest-reporters/__typetests__/*.test.ts",
-    "packages/jest-resolve/__typetests__/*.test.ts",
-    "packages/jest-runner/__typetests__/*.test.ts",
-    "packages/jest-snapshot/__typetests__/*.test.ts",
-    "packages/jest-types/__typetests__/config.test.ts",
-    "packages/jest-types/__typetests__/expect.test.ts",
-    "packages/jest-types/__typetests__/globals.test.ts",
-    "packages/jest-types/__typetests__/jest.test.ts",
-    "packages/jest-worker/__typetests__/*.test.ts"
-  ]
+  "testFileMatch": ["packages/*/__typetests__/*.test.ts"]
```

Comment:
Keeping this line improves performance a bit. This is because root directories other that `packages` get skipped from lookup (for examples, `e2e` good candidate to skip).